src/averageAges.js

```diff
@@ -37,7 +43,17 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people.filter(person => {
+    return withChildren !== undefined
```

Comment:
You can consider adding a default parameter to `function calculateWomenAverageAge(people, withChildren = false) {`, then in check you can only use `return withChildren ? code : code` without undefined